### ISSUE TITLE
Can you let me know the specific steps and the correct rate?

### ISSUE THREAD

**Issue by Hedlen** [chencodeX/Bilinear_CNN_dog_classifi#1]
Hello!
I am repeating your source code, would like to ask you under the final test of the correct rate can be achieved? Because know the correct rate can give me a reference. Can you provide detailed running steps to help me better reproduce? Thank you for your help!
Thanks very much!

**Comment by chencodeX** [comment]
Do you mean the result of that network?
I used tensoflow to create a BCNN and then used pytorch to create many other net.

**Comment by Hedlen** [comment]
Well, yes. I am now running BCNN this, but that Baidu dog data ran out of the results are not very high So would like to ask you under the Baidu dog data test set on the results. . Yeah do a comparison.
And when I running the bcnn_finetuning_predict.py , it will  appear the following error, I don't know why this happen?
`2017-08-14 15:37:29.216704: W tensorflow/core/framework/op_kernel.cc:1158] Not found: Key conv1_1/b not found in checkpoint
2017-08-14 15:37:29.216704: W tensorflow/core/framework/op_kernel.cc:1158] Not found: Key conv4_1/W not found in checkpoint
2017-08-14 15:37:29.216704: W tensorflow/core/framework/op_kernel.cc:1158] Not found: Key conv1_2/b not found in checkpoint
2017-08-14 15:37:29.217077: W tensorflow/core/framework/op_kernel.cc:1158] Not found: Key conv3_3/b not found in checkpoint
`
I look forward to your reply.
Thanks!

**Comment by Hedlen** [comment]
I found that loading model.ckpt file, even the results did not affect. Because I deleted the following few procedures, ran out of the forecast value predict_dog.txt file results are the same. 
`   saver = tf.train.Saver()
    checkpoint_dir = '/home/meteo/zihao.chen/fine-grained-classifi/model8'
    ckpt = tf.train.get_checkpoint_state(checkpoint_dir)
    if ckpt and ckpt.model_checkpoint_path:
        saver.restore(sess, ckpt.model_checkpoint_path)`
I look forward to your reply.
Thank you very much.

**Comment by chencodeX** [comment]
When the model predicts, the model will be loaded before the model is loaded.
You do not have my model, so you did not actually work for this loader.
So in fact it is using the default parameters to predict, so the results will remain the same. And the prediction is very poor.
My best result is 75% accuracy.

**Comment by Hedlen** [comment]
Ok. I have understood it. Although some problems encountered, but also resolved. And the model I also succeeded. But I also found that the correct rate is not very high I still need more time to debug the training。Thank you for your help. Thanks!

**Comment by Hedlen** [comment]
Ok. I have understood it. Although some problems encountered, but also resolved. And the model I also succeeded. But I also found that the correct rate is not very high I still need more time to debug the training. Thank you for your help. Thanks!

**Comment by chencodeX** [comment]
If you want to optimize this BCNN, I can give you some advice.
Bilinear networks have two structures,Symmetrical and asymmetric,Symmetry is the same as two subnetworks,Asymmetry is the difference between the two sub-networks,Here is the realization of the symmetry of the two vgg.

**Comment by chencodeX** [comment]
Vggnet is a very old network, you can change to densenet or resnet, inception and so on.Can also be asymmetric combination with each other。

**Comment by Hedlen** [comment]
Thanks for your advice. Although vggnet is very old network. But the [github project](network.https://github.com/abhaydoke09/Bilinear-CNN-TensorFlow) to achieve the same category with Baidu data set on the correct rate after optimization in 86.4%, but learn from the training of Baidu dog data, the correct rate is very low. There may be a result of a different parameter or data set. I will adopt your advice, with asymmetric two subnetworks as well as using a relatively new.
Thanks again!

**Comment by Hedlen** [comment]
Here is the github project https://github.com/abhaydoke09/Bilinear-CNN-TensorFlow. I believe you have known. Thanks!

**Comment by chencodeX** [comment]
good luck!
If you have a better result, please tell me.

**Comment by Hedlen** [comment]
OK. Thanks!
